htsinfer/cli.py

```diff
@@ -212,6 +212,29 @@ def __call__(
             "the library's source"
         )
     )
+    parser.add_argument(
+        "--library-type-max-distance",
+        dest="lib_type_max_distance",
+        metavar="INT",
+        type=int,
+        default=1000,
+        help=(
+            "upper limit on the difference in the reference "
+            "sequence coordinates between the two mates to be "
+            "considered as coming from a single fragment. "
+        )
+    )
+    parser.add_argument(
+        "--library-type-mates-cutoff",
+        dest="lib_type_mates_cutoff",
+        metavar="FLOAT",
+        type=float,
+        default=0.95,
+        help=(
+            "minimum fraction of mates that can be mapped to compatible loci "
+            "and are considered concordant pairs / all mates. "
+        )
```

Comment:
Maybe add here that these are only used when the lib type cannot be inferred by regex from the seq identifiers